### ISSUE TITLE
Issues always happen while using manager agent mode

### ISSUE THREAD

**Issue by ckgithub2019** [crewAIInc/crewAI#1057]
### I ensure I have defined all of agents and tasks well based on the examples of CrewAI. But I often encounter with such issues while using manager agent mode and hierarchical process. WHY?

**Issue 1:**
I encountered an error while trying to use the tool. This was the error: 2 validation errors for Ask question to coworkerSchema
question
  field required (type=value_error.missing)
context
  field required (type=value_error.missing).
 Tool Ask question to coworker accepts these inputs: Ask a specific question to one of the following coworkers: Manager
The input to this tool should be the coworker, the question you have for them, and ALL necessary context to ask the question properly, they know nothing about the question, so share absolute everything you know, don't reference things but instead explain them.

**Issue 2:**
Action 'Ask question to Research Specialist and Marketing Manager' don't exist, these are the only available Actions:
 Tool Name: Delegate work to coworker(task: str, context: str, coworker: Optional[str] = None, **kwargs)
Tool Description: Delegate a specific task to one of the following coworkers: Research Specialist
The input to this tool should be the coworker, the task you want them to do, and ALL necessary context to execute the task, they know nothing about the task, so share absolute everything you know, don't reference things but instead explain them.
Tool Arguments: {'task': {'title': 'Task', 'type': 'string'}, 'context': {'title': 'Context', 'type': 'string'}, 'coworker': {'title': 'Coworker', 'type': 'string'}, 'kwargs': {'title': 'Kwargs', 'type': 'object'}}
Tool Name: Ask question to coworker(question: str, context: str, coworker: Optional[str] = None, **kwargs)
Tool Description: Ask a specific question to one of the following coworkers: Research Specialist
The input to this tool should be the coworker, the question you have for them, and ALL necessary context to ask the question properly, they know nothing about the question, so share absolute everything you know, don't reference things but instead explain them.
Tool Arguments: {'question': {'title': 'Question', 'type': 'string'}, 'context': {'title': 'Context', 'type': 'string'}, 'coworker': {'title': 'Coworker', 'type': 'string'}, 'kwargs': {'title': 'Kwargs', 'type': 'object'}}

**Comment by syrom** [comment]
Same problem with Langflow:
Error message in terminal:
I encountered an error while trying to use the tool. This was the error: 1 validation error for Delegate work to coworkerSchema
context
  field required (type=value_error.missing).
 Tool Delegate work to coworker accepts these inputs: Delegate a specific task to one of the following coworkers: [Redakteur, Faktenprüfer, Rechercheur]
The input to this tool should be the coworker, the task you want them to do, and ALL necessary context to execute the task, they know nothing about the task, so share absolute everything you know, don't reference things but instead explain them.

Addition: the mentioned 3 Agents [Redakteur, Faktenprüfer, Rechercheur] are all correctly set up. The entire flow works in the end - but seemingly without any delegation to any of the 3 coworkers ?!?
I am using the "Hierachical Crew" Langflow component. on MacOS Sonoma 14.6.1 with Python 3.11.9, crewAI 0.36.1 and crewAI Tools 0.4.26 installed in the environment.
The LLM used is chatGPT-4o - but that doesn't seem to play a role. I tried with Ollama, but had the same effect.
